renovate.json

```diff
@@ -1,32 +1,23 @@
 {
   "$schema": "https://docs.renovatebot.com/renovate-schema.json",
-  "extends": [
-    "config:base"
-  ],
-  "regexManagers": [
-    {
-      "fileMatch": ["(^|/)rockcraft.yaml$"],
-      "description": "Update base image references",
-      "matchStringsStrategy": "any",
-      "matchStrings": ["# renovate: build-base:\\s+(?<depName>[^:]*):(?<currentValue>[^\\s@]*)(@(?<currentDigest>sha256:[0-9a-f]*))?",
-      "# renovate: base:\\s+(?<depName>[^:]*):(?<currentValue>[^\\s@]*)(@(?<currentDigest>sha256:[0-9a-f]*))?"],
-      "datasourceTemplate": "docker",
-      "versioningTemplate": "ubuntu"
-    }
-  ],
+  "extends": ["config:base"],
+  "enabled": true,
   "packageRules": [
     {
-      "enabled": true,
-      "matchDatasources": [
-        "docker"
+      "matchManagers": ["pip_requirements"],
```

Comment:
Perhaps we should use `pep621` as the manager here? https://docs.renovatebot.com/modules/manager/pep621